src/acme/pd/Ticket.java

```diff
@@ -112,35 +116,45 @@ private void updatePath() {
     }
 
     private void calculateQuote() {
-        double milesToTravel = path.getBlocksBetweenHomeandDropoff() * company.getBlocksPerMile();
         double quote = path.getBlocksBetweenHomeandDropoff() * company.getBlockBillingRate().doubleValue();
         this.quotedPrice = new BigDecimal(quote + company.getFlatBillingRate().doubleValue());
     }
 
     private void calcEstimatedTimes() {
-      // TODO estimate times - needs refining
-        // Altered this slightly
-
+      //Lets get down to bussiness.... to defeat the huns!
+      
       double mphCouriers = company.getCourierMilesPerHour();
-      double milesToTravel = company.getBlocksPerMile() / path.getBlocksBetweenHomeandDropoff();
-      double timeToTravel = milesToTravel / mphCouriers;
-      LocalDateTime resultOfCouriersAndMiles = deliveryTime.minus((long)(60*timeToTravel), ChronoUnit.MINUTES);
-      resultOfCouriersAndMiles.plusMinutes(5);
-
-      if(resultOfCouriersAndMiles.isAfter(LocalDateTime.now()))
+      System.out.println("Miles Per Hour: "+ mphCouriers);
+      System.out.println("Blocks per Mile: " + company.getBlocksPerMile());
+      double bphCouriers = mphCouriers * company.getBlocksPerMile();
+      System.out.println("Blocks per Hour: " + bphCouriers);
+      double timeToTravel = path.getBlocksBetweenHomeandDropoff() / bphCouriers;
+      timeToTravel = timeToTravel * 60;
+      System.out.println("Time to Travel before 10 mins added: " + timeToTravel);
```

Comment:
Is the ten minutes for the customer interaction time (like actually dropping off and pickup up packages)?

Reply:
Yes!